spec/rubocop/cop/style/hash_transform_methods_spec.rb

```diff
@@ -0,0 +1,181 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::HashTransformMethods do
+  subject(:cop) { described_class.new }
```

Comment:
Try running tests for each Ruby version with the following changes:

```diff
-RSpec.describe RuboCop::Cop::Style::HashTransformMethods do
-  subject(:cop) { described_class.new }
+RSpec.describe RuboCop::Cop::Style::HashTransformMethods, :config do
+  subject(:cop) { described_class.new(config) }
```

Reply:
Looks like threading `config` through to `described_class.new` was the thing I was missing - thanks!